nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/cache/ConversionCacheHolder.java

```diff
@@ -178,7 +183,12 @@ protected static BlobHolder doGetFromCache(String key) {
                 // skip all negative values
                 CACHE_HITS.addAndGet(Long.MIN_VALUE); // back to 0
             }
-            return cacheEntry.restore();
+            BlobHolder restored = cacheEntry.restore();
+            if (restored.getBlob().getFile().exists()) {
+                return restored;
+            } else {
+                removeFromCache(key);
```

Comment:
Already done by the unique caller `getFromCache` so not appropriate here, no?

Reply:
Yes first wrote it here then "moved" it but forgot to erase the first one !